### PR TITLE
Remove flags from codecov config file

### LINKED ISSUE #3248: Codecov main cov missing
Refresh needed after patch, or perhaps there was a typo in the config?

Screenshot 2022-03-18 at 08 58 01

### PR DESCRIPTION
**Pull request by northwestwitch**
Tentative fix #3248 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Check that issue #3248 is solved in GitHub actions

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DNIL
- [x] tests executed by GitHub actions

### CODE REVIEW

File: codecov.yaml [hunk 1]
```diff
@@ -7,8 +7,6 @@ coverage:
         target: auto
         threshold: 0.3%
         base: auto
-        flags:
```
Comment (dnil) [inline codecov.yaml:10]: 👍

**Review by dnil — APPROVED** [review]
Still tentatively think we would need to clear `branches` as well, but I haven't really tested so...

**Comment by northwestwitch** [comment]
> Still tentatively think we would need to clear `branches` as well, but I haven't really tested so...

Let's see how this run goes!

**Comment by northwestwitch** [comment]
> Still tentatively think we would need to clear `branches` as well, but I haven't really tested so...

You might be right here:

image

**Comment by dnil** [comment]
> > Still tentatively think we would need to clear `branches` as well, but I haven't really tested so...
> 
> You might be right here:
> 
> image

It does say that, but it is working like this. I'd say lets ignore the docs and go with the test until further issues arise?

**Pull request merged**